rust-wasm-id-allocator/distributed-id-allocator/src/compressor.rs

```diff
@@ -42,6 +46,7 @@ impl IdCompressor {
             next_range_base_generation_count: LocalId::from_id(-1).to_generation_count(),
             sessions,
             final_space: FinalSpace::new(),
+            final_id_limit: FinalId::from_id(0),
```

Comment:
My understanding from reviewing the rest of the code is that this is representative of -- and referenced as -- one greater than the numerically largest finalized ID. Do I have that right?